app/controllers/users_controller.rb

```diff
@@ -66,6 +66,6 @@ def set_user
 
     # Only allow a list of trusted parameters through.
     def user_params
-      params.require(:user).permit(:name, :mobile, :email, :password, :password_confirmation, :address)
+      params.require(:user).permit(:name, :mobile, :email, :role, :password, :password_confirmation, :address)
```

Comment:
user should not be able to add role via signup, why we have added this?

Reply:
removed